packages/web/src/components/result/ReactiveList.js

```diff
@@ -420,12 +420,14 @@ class ReactiveList extends Component {
 				filteredResults = filteredResults.filter(item => !ids.includes(item._id));
 			}
 
-			return this.props.onResultStats(
-				this.props.total,
-				this.props.time,
-				this.state.currentPage,
-				[...streamResults, ...filteredResults].length,
-			);
+			const stats = {
+				totalResults: this.props.total,
```

Comment:
We should also return `totalPages` in addition to this.

Reply:
True that will complete all `stats` making that change 👍